javascript-timers/timeout.js

```diff
@@ -0,0 +1,7 @@
+var $title = document.querySelector('.message');
+
+function delayedTitle() {
+  $title.textContent = 'Hello There';
+}
+
+window.addEventListener('DOMContentLoaded', setTimeout(delayedTitle, 2000));
```

Comment:
Again, nothing wrong with this, but you don't need to attach this to an eventListener